src/App.svelte

```diff
@@ -24,6 +24,12 @@
   }
 </script>
 
+<style>
```

Comment:
No need to use a global style here

Reply:
I cannot seem to find a way to apply changes to main without the global any way ?

Reply:
You can use a node around Router that contains the page content